karma.conf.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 
-const webpackConfig = require('./config/webpack.test.js')
+const webpackConfig = require('./config/webpack.testing.js')
```

Comment:
NICE CATCH! 🙇 🎊  (it took a double take for me to....)

Reply:
nitpick: should use es6 import vs require